src/components/material/material-buttons/online/MaterialButtonOnlineExternal.tsx

```diff
@@ -25,13 +25,15 @@ export const getOnlineMaterialType = (
   sourceName: AccessUrl["origin"],
   materialTypes: MaterialType["specific"][]
 ) => {
-  if (sourceName.includes("ereol")) {
+  if (sourceName.toLowerCase().includes("ereol")) {
     return "ebook";
   }
-  if (sourceName.includes("filmstriben")) {
+  if (sourceName.toLowerCase().includes("filmstriben")) {
     return "emovie";
   }
-  if (materialTypes.find((element) => element.includes("lydbog"))) {
+  if (
+    materialTypes.find((element) => element.toLowerCase().includes("lydbog"))
+  ) {
```

Comment:
Don't we have a global place for these material types? I think we should use that

Reply:
I mean, no, but it defo could be added ;)